charts/opentelemetry-collector/values.yaml

```
@@ -307,6 +307,10 @@ service:
   type: ClusterIP
   # type: LoadBalancer
   # loadBalancerIP: 1.2.3.4
+  # loadBalancerSourceRanges:
+  #   - 10.0.0.0/8
+  #   - 172.16.0.0/12
+  #   - 192.168.0.0/16
```

```suggestion
  # loadBalancerSourceRanges: []
```

Instead of giving suggestions on ranges lets only make it clear that this field is supported